iree/tools/run_mlir_main.cc

```diff
@@ -122,6 +122,13 @@ static llvm::cl::list<std::string> input_values_flag{
     llvm::cl::ZeroOrMore,
 };
 
+static llvm::cl::opt<std::string> input_values_file_flag{
+    "input-value-file",
+    llvm::cl::desc(
+        "<input value file>, see run_module_main.cc for input format"),
```

Comment:
"Provides a file for input shapes and optional values (see ... for details)" ?

Actually I think we might what to put the format spec in https://google.github.io/iree/DeveloperOverview so we can point to a URL here. Source code is not always there. :)

Reply:
Cool, I'll follow on this once all the tools get support of input file.